hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java

```diff
@@ -528,8 +529,8 @@ public static String getCodePoint(final StatusFontIcon statusFontIcon) {
      * @see java.util.Locale#getDefault()
      */
     public static Locale getCurrentLocale() {
-        final UI currentUI = UI.getCurrent();
-        return currentUI == null ? Locale.getDefault() : currentUI.getLocale();
+        final Page page = Page.getCurrent();
+        return page == null ? Locale.getDefault() : page.getWebBrowser().getLocale();
```

Comment:
pls double-check if defaultLocale config property is still accepted...

Reply:
if prop is set, it takes precedence... otherwise browser locale, otherwise system locale